docs/guides/genomics-platform/requesting-data/about-our-data.md

```diff
@@ -23,19 +22,45 @@ We provide gVCF files produced by the [Microsoft Genomics Service][msgen]. gVCF
+
+#### Variant Filtering
+
+We only consider variants for reporting with "VALID", "LIKELY_VALID", or "PUTATIVE" designations (three highest confidence designations) from the variant validation status assessment. This determination is stored in a patient somatic VCF file under the `validation_status` info tag. Note that variants designated "PUTATIVE" are filtered out if they were not computationally determined to be of high quality according to an in-house metric. Any variants which were not manually reviewed by an analyst (no 'validation_status') require this high quality designation to pass filtering. When variants are not manually reviewed, they receive a `validation_status` of "NA".
+
+#### Further notes on patient tumor sample VCF files
 
+As part of our post-processing pipeline, some variants are annotated with the results of a variety of pathogenicity predictive algorithms. We include those results in the VCFs when they are present. Note that they were performed on the HG19 coordinates. Algorithms include [Likelihood Ratio Test (LRT)][lrt], [MutationAssessor][assessor], [MutationTaster][taster], and [FATHMM][fathmm].
+
+Bambino represents indels in a format incompatible with the VCF specification. When we convert variants to VCFs, we store the original Bambino representation in the `bambino_representation` tag of the info field. Note that this entry is always in HG19 coordinates.
+
+Where possible, read evidence for each variant was determined across each sequencing data type available for the associated patient tumor normal pair. These can include WGS, WES and RNAseq data.
+
+The newly created HG19 coordinate VCFs are lifted over using [Picard `LiftoverVcf`][picard]. [`hg19ToHg38.over.chain`][chain] is used as the chain file and `GRCh38_no_alt.fa` is used as the reference. [`bcftools annotate`][bcftools] is used to document the reference file and exact liftover command used in the VCF's header. Note that HG19 alleles and coordinates are stored by Picard in the `OriginalAlleles`, `OriginalContig`, and `OriginalStart` info tags within the final VCF file. Please also note that there is a bug in the version of Picard we used (version 2.18.29) which rarely truncates some of the VCF's genotype fields. These are fields which we use to store read counts and depths for each sequencing type (WGS, WES, RNAseq). A [bug report][bug_report] has been filed with Picard. In the current version of the pipeline we recover the dropped entries from the original HG19 VCF. While searching for and correcting these entries, we also reorder the FORMAT and genotype fields to a logical ordering, as opposed to the alphabetical output order of Picard.
```

Comment:
We need to address that hg38 includes some reference sequences (alt contigs, etc) that our reference does not. How do we handle those sequences, and why is it okay that we use slightly different versions there?

Reply:
For reference, here are the sequences included in the hg38 chain reference and how many associated chains for each:

```
 184 chr1
 102 chr10
  11 chr11
   2 chr11_gl000202_random
  42 chr12
   1 chr13
  80 chr14
  19 chr15
  30 chr16
  28 chr17
   3 chr17_ctg5_hap1
   1 chr17_gl000203_random
   1 chr17_gl000204_random
   3 chr17_gl000205_random
   3 chr17_gl000206_random
   3 chr18
   1 chr18_gl000207_random
  36 chr19
   1 chr19_gl000208_random
  13 chr19_gl000209_random
   1 chr1_gl000191_random
   4 chr1_gl000192_random
 120 chr2
   2 chr20
   2 chr21
   2 chr21_gl000210_random
  20 chr22
  34 chr3
   9 chr4
   2 chr4_ctg9_hap1
   1 chr4_gl000193_random
   1 chr4_gl000194_random
   2 chr5
  17 chr6
   2 chr6_apd_hap1
   2 chr6_cox_hap2
   2 chr6_dbb_hap3
   2 chr6_mann_hap4
   2 chr6_mcf_hap5
   2 chr6_qbl_hap6
   2 chr6_ssto_hap7
  49 chr7
   1 chr7_gl000195_random
  36 chr8
   1 chr8_gl000196_random
   2 chr8_gl000197_random
 135 chr9
   1 chr9_gl000198_random
   1 chr9_gl000199_random
   1 chr9_gl000200_random
   1 chr9_gl000201_random
   1 chrM
   1 chrUn_gl000211
   1 chrUn_gl000212
   1 chrUn_gl000213
   1 chrUn_gl000214
   2 chrUn_gl000215
   1 chrUn_gl000216
   1 chrUn_gl000217
   1 chrUn_gl000218
   1 chrUn_gl000219
   1 chrUn_gl000220
   1 chrUn_gl000221
   3 chrUn_gl000222
   1 chrUn_gl000223
   1 chrUn_gl000224
   1 chrUn_gl000225
   1 chrUn_gl000226
   7 chrUn_gl000227
  16 chrUn_gl000228
   1 chrUn_gl000229
   1 chrUn_gl000230
   1 chrUn_gl000231
   1 chrUn_gl000232
   1 chrUn_gl000233
   1 chrUn_gl000234
   1 chrUn_gl000235
   1 chrUn_gl000236
   1 chrUn_gl000237
   1 chrUn_gl000238
   1 chrUn_gl000239
   1 chrUn_gl000240
   1 chrUn_gl000241
   1 chrUn_gl000242
   1 chrUn_gl000243
   1 chrUn_gl000244
   1 chrUn_gl000245
   1 chrUn_gl000246
   1 chrUn_gl000247
   1 chrUn_gl000248
   1 chrUn_gl000249
 166 chrX
  25 chrY
```

Reply:
Is it suffice to say that if Picard's look up in the chain is to a contig not present in GRCh38_no_alt, it considers the variant unmappable? We don't do any special handling for those cases, Picard handles it for us.